dev/com.ibm.ws.logstash.collector_fat/publish/files/startLogstash.bat

```diff
@@ -0,0 +1 @@
+lib\LibertyFATTestFiles\logstash-2.1.1\bin\logstash.bat -f lib\LibertyFATTestFiles\logstash-2.1.1\logstash.conf --verbose > lib\LibertyFATTestFiles\logstash_console.txt
```

Comment:
This file is probably not used anymore as it is referring to an old version of Logstash.